internal/pkg/ecs/ecs.go

```diff
@@ -180,13 +184,65 @@ func (c Client) ListActiveDefaultClusterTasks(filter ListTasksFilter) ([]*ecs.Ta
 	})
 }
 
+// StopWorkloadTasks stops all tasks in the given application, enviornment, and workload.
+func (c Client) StopWorkloadTasks(app, env, workload string) error {
+	tdFamilyName := fmt.Sprintf(fmtWorkloadTaskDefinitionFamily, app, env, workload)
+	return c.stopTasks(app, env, tdFamilyName, false)
+}
+
+// StopOneOffTasks stops all one-off tasks in the given application and environment with the family name.
+func (c Client) StopOneOffTasks(app, env, family string) error {
+	tdFamilyName := fmt.Sprintf(fmtTaskTaskDefinitionFamily, family)
+	return c.stopTasks(app, env, tdFamilyName, true)
+}
+
+// stopTasks stops all tasks in the given application and environment in the given family.
+func (c Client) stopTasks(app, env, family string, oneOffOnly bool) error {
```

Comment:
nit: 
```suggestion
func (c Client) stopTasks(app, env, filter ListTasksFilter) error {
```
and this way the caller can do:
```go
return c.stopTasks(app, env, ListTasksFilter{
    TaskGroup: fmt.Sprintf(fmtWorkloadTaskDefinitionFamily, app, env, workload),
})
// or
c.stopTasks(app, env,  ListTasksFilter{
    TaskGroup:fmt.Sprintf(fmtTaskTaskDefinitionFamily, family),
    CopilotOnly: true,
})

```